### DISCUSSION TITLE
Feature Flags support

### DISCUSSION THREAD

**Discussion by Neha** [AmazonAppDev/react-native-multi-tv-app-sample#37] [Ideas]
# Problem?
For multi platform tv app, it would be common to have a feature for one platform and which is not supported for other. Eg: Back Button. For this we can use "feature Flag". 

# Solution

Instead of using Third party platform, we can create a environment and platform config to control features.

```
// featureFlagsConfig.js

const featureFlags = {
  backButton: true,  // Enables backButton
  webBanner: false,  // Enables banner in web
  tvOSAdsPanel: true, // Enables ads panel tvOS
};
export default featureFlags;
```

this config can be used at local to enable and disable features and based on OS too.

**Comment by giolaq** [comment]
Ok, so more a configuration parameter than a feature flag? Feature flags are enabled/disabled at runtime ( usually via a server/cloud ). I understand in this proposed implementation this is happening at build time.
